README.md

```diff
@@ -168,6 +168,21 @@ otherwise:
 $ kubectl apply -k deploy/k8s/namespace/
 ```
 
+## Authentication
+
+Hawtio Online currently supports two authentication modes: `oauth` and `form`, which is configured through `HAWTIO_ONLINE_AUTH` environment variable on Deployment.
+
+| Mode | Description |
+| ---- | ----------- |
+| oauth | Authenticates through [OpenShift OAuth server](https://docs.openshift.com/container-platform/4.9/authentication/understanding-authentication.html). It is available only on OpenShift. |
+| form | Authenticates with a [ServiceAccount](#creating-user-for-form-authentication) throught the Hawtio login form. |
+
+### Creating user for Form authentication
```

Comment:
It may be more correct to say that using a token from a ServiceAccount is one option, as there are other mechanisms that could be used like: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-token-file. So I would suggest, to emphasise that this is one example, but any bearer token can be used.

Reply:
Thanks!  Updated.